akka-stream/src/main/boilerplate/akka/stream/impl/GenJunctions.scala.template

```diff
@@ -33,4 +33,24 @@ private[akka] object GenJunctions {
   }#
   ]
 
+  sealed trait UnzipWithModule {
+      /** Allows hiding the boilerplate Props creation from the materializer */
+      def props(settings: ActorFlowMaterializerSettings): Props
+    }
+
+  [2..20#
+  final case class UnzipWith1Module[B, [#A1#]](
+      shape: FanOutShape1[B, [#A1#]],
+      f: B ⇒ ([#A1#]),
+      override val attributes: OperationAttributes = name("unzipWith1")) extends FanOutModule with UnzipWithModule {
+
+      override def withAttributes(attr: OperationAttributes): Module = copy(attributes = attr)
+
+      override def carbonCopy: Module = UnzipWith1Module(shape.deepCopy(), f, attributes)
+
+      override def props(settings: ActorFlowMaterializerSettings): Props =
+        Props(new Unzip1With(settings, f.asInstanceOf[Function##1[Any, ([#Any#])]]))
+    }#
+    ]
+
```

Comment:
Good. Needs to be mirrored in `javadsl` - see here for ZipWith: https://github.com/akka/akka/blob/release-2.3-dev/akka-stream/src/main/boilerplate/akka/stream/javadsl/ZipWith.scala.template